omniledger/service/contracts.go

```diff
@@ -10,14 +10,14 @@ import (
 	"github.com/dedis/protobuf"
 )
 
-// zeroNonce is 32 bytes of zeroes.
-var zeroNonce = Nonce([32]byte{})
+// ZeroNonce is 32 bytes of zeroes.
+var ZeroNonce = Nonce([32]byte{})
 
-// zeroSubID is 32 bytes of zeroes as a subid for storing Darcs.
-var zeroSubID = SubID([32]byte{})
+// ZeroSubID is 32 bytes of zeroes as a subid for storing Darcs.
+var ZeroSubID = SubID([32]byte{})
```

Comment:
I just unexported these on purpose, following the go standard of expose the least possible, and also that uninitialized structs should be useful, which they are because instance id zero means something useful.

So, are you sure you want these exported? No big deal for me, just thought it unnecessary.

Reply:
We could change it by having a `NewDarcInstanceID` and then un-exporting `ZeroSubID`. What about that?

Reply:
In fact `SubID{}` is so much fun, that `NewDarcInstanceID` is just `InstanceID{darc.ID(), SubID{}}`. So no method needed.